setup.cfg

```diff
@@ -116,7 +115,6 @@ asdf_schema_tests_enabled = true
 asdf_schema_root = sunpy/io/special/asdf/schemas/
 mpl-results-path = figure_test_images
 mpl-use-full-test-name = True
-asyncio_mode = auto
```

Comment:
I think this ini option was added by mistake because someone had an extra non-sunpy pytest plugin installed which gave a warning if this ini option was missing.

Reply:
Ill remove it again.